packages/extensions/src/list/style.css

```
@@ -1 +1,123 @@
+      won't affect the counter of the parent list.
+  
+      See also https://github.com/ocavue/prosemirror-flat-list/issues/23
+      */
+    & > * {
+      contain: style;
+    }
+
+    &::before {
+      position: absolute;
+      right: calc(100% - 1.5em);
+      right: calc(100% - 1lh);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

right is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)